gusto/transport_equation.py

```diff
@@ -212,11 +212,37 @@ class EmbeddedDGAdvection(AdvectionEquation):
               constructed for you.
     :arg solver_params: (optional) dictionary of solver parameters to pass to the
                         linear solver.
+    :arg recovered_spaces: A list or tuple of function spaces to be used for the
+                           recovered space advection method. The must be three
+                           spaces, indexed in the following order:
+                           [0]: the embedded space in which the advection takes place.
+                           [1]: the continuous recovered space.
+                           [2]: a broken or discontinuous version of the original space.
```

Comment:
Can you not make `2` automatically from `0`?  `Vbroken = FunctionSpace(V0.ufl_domain(), BrokenElement(V0.ufl_element())` I think?

Reply:
Yeah I had thought about doing this so that it automatically chooses Vbroken. I wasn't for now though because I wanted to be able to choose different Vbroken so I was putting it in by hand!